README.md

```diff
@@ -2,7 +2,9 @@ AsseticBundle
 =============
 
 The `AsseticBundle` provides integration of the [Assetic](https://github.com/kriswallsmith/assetic)
-library into the Symfony2 framework.
+library into the Symfony framework (up to Symfony 3.4).
+
+**CAUTION**: As of Symfony 4.0, Symfony applications should use Webpack Encore. 
```

Comment:
Sorry, I should have commented that with the other remark: Should we also link "Webpack Encore" with https://symfony.com/doc/current/frontend.html?

Reply:
Done :)